lib/spack/spack/package.py

```diff
@@ -2694,45 +2729,56 @@ def rpath_args(self):
         """
         return " ".join("-Wl,-rpath,%s" % p for p in self.rpath)
 
+    def _run_test_callbacks(self, method_names, build_time=False):
+        """Tries to call all of the listed methods, returning immediately
+           if the list is None."""
+        if method_names is None:
+            return
+
+        callback_desc = 'build' if build_time else 'install'
+        fail_fast = spack.config.get('config:fail_fast', False)
+        externals, verbose = False, False
+
+        with self._setup_test(verbose, externals) as logger:
```

Comment:
```suggestion

        with self._setup_test(verbose=False, externals=False) as logger:
```

I think this is easier to read as well, let me know if you disagree